File: Math & Geometry/python/convertIntToExcelColumn.py
def convertToTitle(self, columnNumber: int) -> str:
    # O(log n) => Log base 26 of n
    title = ""
    # convert from base 10 to base 26
    # because we start at 1, we need to subtract 1 from columnNumber before we do the % and // operators
    while columnNumber > 0:
        offset = (columnNumber - 1) % 26
        title += chr(ord('A') + offset)
        columnNumber = (columnNumber - 1) // 26

    return title[::-1]

File: Math & Geometry/cpp/convertIntToExcelColumn.cpp
#include<string>
#include<algorithm>
using namespace std;

string convertToTitle(int columnNumber) {
    string title = "";
    int offset = 0;
    while(columnNumber > 0){
        offset = (columnNumber - 1) % 26;
        title += 'A' + offset;
        columnNumber = (columnNumber - 1) / 26;
    }
    reverse(title.begin(), title.end());
    return title;
}